LastInfoSec/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7.4-alpine3.10
```

Comment:
Please use the current standard `python:3.7-alpine`

Reply:
Agree.

Reply:
Fixed :)